project_euler/euler_prob_4/prob4.js

```diff
@@ -0,0 +1,40 @@
+
+// make a function that checks to see if the multiple of two 3-diget numbers is a palindrome, find the largest one
+// https://projecteuler.net/problem=4
+
+function reverse_a_number(n){
+  n = n + "";
+  return n.split("").reverse().join("");
+}
+
+function three_diget_numbers(){
+  threedigetnumbers = []
+  for (i = 999; i > 99; i-=1){
+    threedigetnumbers.push(i)
+}
+  return threedigetnumbers
+}
+
+function palindrome_array_creator(array){
+  var palindromearray = []
```

Comment:
Could you re-factor this so your three_diget_numbers() function gave you an array of products instead of just an array of numbers, instead of doing it all in this function?